ultralytics/utils/checks.py

```diff
@@ -333,18 +335,19 @@ def check_font(font="Arial.ttf"):
         return file
 
 
-def check_python(minimum: str = "3.8.0", hard: bool = True) -> bool:
+def check_python(minimum: str = "3.8.0", hard: bool = True, verbose: bool = True) -> bool:
```

Comment:
@glenn-jocher Probably should be `False` by default. It's causing a spam of logs.

![image](https://github.com/user-attachments/assets/a214fabf-47eb-4423-bfeb-7a886dd52f30)